src/data/faq_de.json

```diff
@@ -850,7 +943,7 @@
                                 "anchor": "admission_policy",
                                 "active": false,
                                 "textblock": [
-                                    "<b>Aktualisiert am 30. März 2022: </b> Seit Version 2.18 zeigt Ihnen die Corona-Warn-App den Status-Nachweis 2G+ auch für „frisch“-geimpfte, “frisch”-genesene und „geboosterte“ Benutzer an, wenn die entsprechenden digitalen COVID-Zertifikate in der App abgelegt werden und auch auf den gleichen Namen sowie Geburtsdatum ausgestellt worden sind.",
+                                    "<b>Aktualisiert am 23. September 2022: </b> <br> <span style='color:red;font-size: 24px;font-weight: bold;'>!</span><b> Der Status-Nachweis (3G, 3G+, 2G, 2G+) ist zurzeit nicht relevant und wird daher nicht von der App ausgewiesen</b><hr>Seit Version 2.18 zeigt Ihnen die Corona-Warn-App den Status-Nachweis 2G+ auch für „frisch“-geimpfte, “frisch”-genesene und „geboosterte“ Benutzer an, wenn die entsprechenden digitalen COVID-Zertifikate in der App abgelegt werden und auch auf den gleichen Namen sowie Geburtsdatum ausgestellt worden sind.",

```

Comment:
The FAQ should say from which version the suppression of 3G, 3G+, 2G, 2G+ indicators started (2.27), otherwise the rest of the text mentions several other versions (2.16, 2.18 & 2.20) so it is unclear.

Reply:
We'll insert the release number 2.27 in the "Aktualisiert" (respectively "Updated") statement.

Reply:
I think this was already mentioned in the very beginning of new faq article:
![image](https://user-images.githubusercontent.com/69585203/192301339-5b87e2c3-bba4-45e2-a806-bf652f1ab4d0.png)

Reply:
...and:
![image](https://user-images.githubusercontent.com/69585203/192302317-932c5121-5295-448f-970f-a2f33ce87014.png)

Reply:
@GisoSchroederSAP New FAQ entries hold the information but the point is the removal of the e.g. 2G badge.

Reply:
@GisoSchroederSAP 

My comment was about the article "#admission_policy". Your response refers to the other article "#mask_rules" which is further down.

Reply:
CCL Version 3.3.1 is already deploying to CWA 2.26 and is not dependent on an app update (to 2.27). I did not know this when I made my original comment. CCL is what  decides whether there is a 3G / 2G status to be displayed or not.

CCL config updates generally take up to 24 hours to deploy.